### DISCUSSION TITLE
Remaking index.html

### DISCUSSION THREAD

**Discussion by justin-richling** [NCAR/ADF#216] [Ideas]
Working with Will Chapman on an ADF issue, a conversation arose about whether the index.html file should be re-created every time the ADF is run.

I think this is a good move because if the user changes the ADF (adds/removes variables, plots, etc) the index.html file doesn't reflect this and can cause a deal of confusion if the user is unaware that the html file does not get updated. Currently the user would have to delete the index.html file and re-run the ADF to get the html file updated.

Since it is a fairly basic file, re-creating it shouldn't be too big of a deal in my opinion and could help reduce ADF missteps.

Any thoughts?

**Comment by bitterbark** [comment]
Agree that it should be re-written every time!
